tests/custom/flatten_lex/sets/set_of_int/stdout.expected

```diff
@@ -0,0 +1,66 @@
+language ESSENCE' 1.0
+
+find A_Occurrence: matrix indexed by [int(1..10)] of bool
+find B_Occurrence: matrix indexed by [int(1..10)] of bool
+branching on [A_Occurrence, B_Occurrence]
+such that [-toInt(A_Occurrence[q3]) | q3 : int(1..10)] <lex [-toInt(B_Occurrence[q4]) | q4 : int(1..10)]
+
+$ Conjure's
```

Comment:
I really don't like committing the comment lines as part of test results. Use `head -n6` to filter then out?